src/main/java/in/twizmwaz/cardinal/module/modules/permissions/PermissionModule.java

```diff
@@ -172,8 +193,9 @@ public void onPlayerNameUpdate(PlayerNameUpdateEvent event) {
             stars += ChatColor.GOLD + star;
         } else if (isMod(event.getPlayer().getUniqueId())) {
             stars += ChatColor.RED + star;
-        }
-        if (devs.contains(event.getPlayer().getUniqueId())) {
```

Comment:
Why did you remove this `if` statement? That makes it so someone can't have both an op star and a dev star (or any other two stars).